src/main/resources/views/CommercialVehicles/MSVA30.hbs

```diff
@@ -76,12 +75,12 @@
             {{#if msvaData.requiredStandards}}
                 <ul class="required-standard-list">
                     {{#each msvaData.requiredStandards}}
-                        <li>
+                        <li class= "required-standard-list-item">
```

Comment:
Last AC is not being met:

There must not be a page break within a section defect block of text

It looks like it is page breaking in the middle of a defect